File: Iniciante/1005 - Média 1/JavaScript/1005.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const a = parseFloat(lines.shift());
  const b = parseFloat(lines.shift());
  const WEIGHT_A = 3.5;
  const WEIGHT_B = 7.5;
  const TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B;
  const average = (a * WEIGHT_A + b * WEIGHT_B) / TOTAL_WEIGHT;
  console.log(`MEDIA = ${average.toFixed(5)}`);
}

main();

File: Iniciante/1005 - Média 1/Python/1005.py
def main():
    A = float(input())
    B = float(input())
    WEIGHT_A = 3.5
    WEIGHT_B = 7.5
    TOTAL_WEIGHT = WEIGHT_A + WEIGHT_B
    average = (A * WEIGHT_A + B * WEIGHT_B) / TOTAL_WEIGHT
    print(f"MEDIA = {average:.5f}")


if __name__ == "__main__":
    main()